### ISSUE TITLE
Leshan/Wakaama and Test Object 1024

### ISSUE THREAD

**Issue by sbernard31** [eclipse/wakaama#309]
Hi,
   At Leshan, we regulary face LWM2M "newb" who doesn't understand why the 1024 Test object of wakaama is not well detected in Leshan.

   Should we try to solve this together ? I think about 2 solutions :
1) Adding the "Wakaama Test object" model in Leshan demo. In that case we replace the 1024 ID by a `x-label` one ([26241 - 32768 Range is privately reused and there is no need for registration](http://www.openmobilealliance.org/wp/OMNA/LwM2M/LwM2MRegistry.html)).
2) Replace Test object by one or several already defined objects. (like IPSO objects)

  WDYT ? Any preferences ?

**Comment by dnav** [comment]
1/ is the better IMHO. Let's bump it to 31024.
I'll take care of this.

**Comment by sbernard31** [comment]
Ok I will add the model in Leshan Server Demo.

The description is correct ? 
```
 *                 Multiple
 * Object |  ID  | Instances | Mandatoty |
 *  Test  | 31024 |    Yes    |    No     |
 *
 *  Resources:
 *              Supported    Multiple
 *  Name | ID | Operations | Instances | Mandatory |  Type   | Range | Units |      Description      |
 *  test |  1 |    R/W     |    No     |    Yes    | Integer | 0-255 |       |                       |
 *  exec |  2 |     E      |    No     |    Yes    |         |       |       |                       |
 *  dec  |  3 |    R/W     |    No     |    Yes    |  Float  |       |       |                       |
 *  sig  |  4 |    R/W     |    No     |    Yes    | Integer |       |       | 16-bit signed integer |
```

**Comment by dnav** [comment]
Resource 4 is optional.
Maybe it would be interesting to not provide the model in Leshan as an example of what happens if the Object layout is not shared between the client and the server ?

**Comment by sbernard31** [comment]
I suppose the user first step is to make examples works and this is maybe frustrating to be blocked at this first step. Adding the model could change this.
Adding their own custom objects is a second step where they will face this missing model issue ?

**Comment by sbernard31** [comment]
I tested your fix (fd037a7) with https://github.com/eclipse/leshan/pull/359, it sounds good to me.

**Comment by dnav** [comment]
Fixed by #310
